routes/api.php

```diff
@@ -35,3 +42,29 @@
         Route::post('/', StoreUserController::class);
         Route::delete('/{user}', DeleteUserController::class);
     });
+
+/*
+|--------------------------------------------------------------------------
+| Employees Routes
+|--------------------------------------------------------------------------
+*/
+Route::prefix('employees')
+    ->middleware([])
+    ->group(static function () {
+        Route::get('/', ListEmployeeController::class);
+        Route::post('/', StoreEmployeeController::class)->name('employees');
+});
```

Comment:
```suggestion
Route::prefix('employees')
    ->name('employees.')
    ->middleware([])
    ->group(static function () {
        Route::get('/', ListEmployeeController::class)->name('list');
        Route::post('/', StoreEmployeeController::class)->name('store');
});
```
I use to do this instead because it allows us to reduce significantly the name length in the endpoint as every endpoint inside the group will have the prefix name 'employees.' with this
This is my personal recommendation; feel free to ignore it if you don't like it.